content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,448 @@
+---
+title: VMAX
+menuTitle: VMAX 
+description: "VMAX Support Matrix"
+weight: 19
+disableToc: false
+tags: ["support matrix", "delfin", "drivers", "dell emc"]
+---
+
+#### Model and versions supported
+ 
+ <table>
+     <thead>
+         <tr>
+             <th>Models</th>
+             <th>Firmware Versions</th>
+         </tr>
+     </thead>
+     <tbody>
+         <tr>
+            <td>VMAX 100K, VMAX 200K,VMAX 400K</td>
+            <td>Unisphere 9.0 or later</td>
+         </tr>
+         <tr>
+             <td></td>
+             <td></td>
+         </tr>
+     </tbody>
+ </table>
+ 
+ ### Access information to register storage
```

Comment:
This does not appear as a heading in the deploy preview.
Could you please check again.

Reply:
This is a screenshot from the deploy preview.
![image](https://user-images.githubusercontent.com/19162717/108018003-79dff080-703c-11eb-9cfa-0696801b8582.png)

Reply:
corrected